frontend/prebuild/src/app/entity/player.tooltip.ts

```diff
@@ -1,5 +1,6 @@
 import { Shape, Text } from 'createjs-module';
 import { Player } from './player';
+import { GameComponent } from '../game/game.component';
```

Comment:
This import is causing a circular dependency warning for me. Perhaps we can split out constants to a separate component later

Reply:
On my list for the refactor I'm working on